File: solutions/java/lasagna/2/src/main/java/Lasagna.java
public class Lasagna {
    public int expectedMinutesInOven(){
        return 40;
    }
    public int remainingMinutesInOven(int timePassed){
        return expectedMinutesInOven() - timePassed;
    }
    public int preparationTimeInMinutes(int layers){
        return 2 * layers;
    }
    public int totalTimeInMinutes(int layers, int timePassed){
        return preparationTimeInMinutes(layers) + timePassed;
    }
}

File: solutions/swift/lasagna/1/Sources/Lasagna/Lasagna.swift
let expectedMinutesInOven: Int = 40

func remainingMinutesInOven(elapsedMinutes timePassed: Int) -> Int {
  return expectedMinutesInOven - timePassed;
}

func preparationTimeInMinutes(layers : Int) -> Int {
  return layers * 2;
}

func totalTimeInMinutes(layers: Int, elapsedMinutes: Int) -> Int {
  return preparationTimeInMinutes(layers: layers) + elapsedMinutes;
}